src/cache.rs

```diff
@@ -100,124 +138,167 @@ pub fn id() -> usize {
 /// Comemo's cache is thread-local, meaning that this only evicts this thread's
 /// cache.
 pub fn evict(max_age: usize) {
-    CACHE.with(|cache| {
-        let mut cache = cache.borrow_mut();
-        cache.map.retain(|_, entries| {
-            entries.retain_mut(|entry| {
-                entry.age += 1;
-                entry.age <= max_age
-            });
-            !entries.is_empty()
-        });
-    });
-    ACCELERATOR.with(|accelerator| accelerator.borrow_mut().clear());
+    CACHES.read().iter().for_each(|fun| fun(max_age));
+
+    // Evict all accelerators.
+    let mut accelerators = ACCELERATORS.write();
+
+    // Force the ID to be loaded.
+    let id = ID.load(Ordering::Acquire);
+
+    // Update the offset.
+    accelerators.0 = id;
```

Comment:
```suggestion
    // Update the offset.
    accelerators.0 = ID.load(Ordering::Acquire)
```
I don't understand the meaning of the "force" in the comment "Force the ID to be loaded.".

Reply:
It was with the old way it was working where it **HAD TO** do an `Acquire` atomic operation before continuing (since I wasn't using the id afterwards)